shell/platform/fuchsia/dart_runner/dart_runner.cc

```diff
@@ -248,11 +351,34 @@ void DartRunner::Start(
     fuchsia::component::runner::ComponentStartInfo start_info,
     fidl::InterfaceRequest<fuchsia::component::runner::ComponentController>
         controller) {
-  std::string url_copy = start_info.resolved_url();
-  TRACE_EVENT1("dart", "Start", "url", url_copy.c_str());
-  std::thread thread(RunApplicationV2, this, std::move(start_info),
-                     context_->svc(), std::move(controller));
-  thread.detach();
+  /// Parse the program field of the component's cml and check if it is a test
```

Comment:
/// comments are only for documentation on classes/methods not regular comments.